rust/examples/dynamic_types.rs

```diff
@@ -20,9 +20,9 @@ use std::sync::Arc;
 
 extern crate arrow;
 
-use arrow::array::*;
-use arrow::datatypes::*;
-use arrow::record_batch::*;
+use arrow::arrow::array::*;
+use arrow::arrow::datatypes::*;
+use arrow::arrow::record_batch::*;
```

Comment:
The repeated namespace isn't ideal ... this is because we are building both arrow and parquet as a single crate? Maybe we should keep it as separate crates?

Reply:
What do we think about adding `parquet` alongside the `csv` as a sub module if building as a single crate?

Reply:
Making them separate crate adds extra dependency - say if I want to use a new feature in arrow for my parquet arrow reader, I have to wait for the next crate release, which is undesirable. 

Yes I'm fine by keeping `parquet` alongside `csv` and keep the `arrow` code top level as they are right now. This can solve the repeated namespace issue.